Assets/Scripts/Models/Inventory.cs

```diff
@@ -23,6 +21,7 @@ public class Inventory : IXmlSerializable, ISelectable, IContextActionProvider
     public string objectType = "Steel Plate";
     public int maxStackSize = 50;
     public float basePrice = 1f;
+    public string category = "inv_cat_none";
```

Comment:
I just follow the document, everything was public field, I added one public field.
But I can change it :) I will when I have access to my computer (and time)